src/crescendo.ts

```diff
@@ -58,7 +58,6 @@ export class Crescendo extends Note {
   }
 
   protected decrescendo: boolean;
-  protected height: number;
```

Comment:
I see height is removed in a few classes. Do we not need this anymore?

Reply:
These classes derive from Element which already define height (this is like width above). The compiler was not complaining with variables but it does complain when the items are accessors in the base class.